crm_action/models/crm_action.py

```diff
@@ -55,18 +41,20 @@ def default_action_type(self):
         action_types = self.search_action_types()
         return action_types and action_types[0].id or False
 
+    lead_id = fields.Many2one('crm.lead', string='Lead', ondelete='cascade')
+    partner_id = fields.Many2one('res.partner', string='Customer')
+    date = fields.Date(
+        'Date', required=True, default=fields.Date.context_today)
+    user_id = fields.Many2one(
+        'res.users', string='User', required=True,
+        default=lambda self: self.env.user)
```

Comment:
Put field definitions above methods.

If you need a method defined, then use a lambda as you do here, or the method name in places where the ORM allows it.